sleuth/README.md

```diff
@@ -0,0 +1,58 @@
+## Overview
+
+Sleuth is a deployment tracking tool that enables you to track software deployments through your complete DevOps stack. With a Datadog integration, Sleuth provides you with insightul, meaningful and actionable real-time data that enable you and your team to see, with clarity, the impact of the changes you make to your code.
```

Comment:
```suggestion
[Sleuth](https://app.sleuth.io) is a deployment tracking tool that enables you to track software deployments through your complete DevOps stack. With a Datadog integration, Sleuth provides you with insightful, meaningful and actionable real-time data that enable you and your team to see, with clarity, the impact of the changes you make to your code.
```

Can we add a link to Sleuth?

Reply:
Thanks for the catch!  Yes, a link to Sleuth can be added. Please use **https://www.sleuth.io** instead of the app URL.